From fd2aa6a4a3c9abb223abcf5b226deae5bd71182a Mon Sep 17 00:00:00 2001
From: Chen Meng J <meng.j.chen@intel.com>
Date: Thu, 25 Oct 2018 16:20:44 +0800
Subject: [PATCH 001/292] mm: export some vm_area APIs

Message for Open Source:
These APIs are needed in DMA mapping API implementation
as a module. Device specific IOMMUs with associated DMA
mapping implementations should be buildable as modules.

Message for Internal:
kernel upgrade to 4.20

Change-Id: I46bf3823345ae0bd9cd437ea25b6b29a4b727126
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 mm/vmalloc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mm/vmalloc.c b/mm/vmalloc.c
index 871e41c55e23..43158f787bcd 100644
--- a/mm/vmalloc.c
+++ b/mm/vmalloc.c
@@ -1461,6 +1461,7 @@ struct vm_struct *find_vm_area(const void *addr)
 
 	return NULL;
 }
+EXPORT_SYMBOL_GPL(find_vm_area);
 
 /**
  *	remove_vm_area  -  find and remove a continuous kernel virtual area
-- 
2.21.0

